src/components/FindWallet/WalletTable/index.tsx

```diff
@@ -328,9 +336,11 @@ const WalletTable = ({
               <Wallet
                 // Make wallets more info section open on 'Enter'
                 onKeyUp={(e) => {
-                  if (e.key === "Enter") showMoreInfo(wallet)
+                  if (e.key === "Enter") showMoreInfo(wallet, e)
+                }}
+                onClick={(e) => {
+                  showMoreInfo(wallet, e)
```

Comment:
Hey @Dharmik79, thanks for taking a stab at this! I don't think we need to get into looking at class names here... I think this could be solved with a one-liner to just check if the event target is an "A" tag, and if so, return early to avoid toggling "showMoreInfo"

```suggestion
                onClick={(e) => {
                  if ((e.target as HTMLTableRowElement).tagName === "A") return
                  showMoreInfo(wallet)
```

Reply:
Hey @wackerow, The solution will work only when the button is pressed, but by clicking the SVG icon within the button unintentionally triggers both the 'showMoreInfo' function and opens a new page simultaneously. To resolve this, I have used 'classList' to specifically address and prevent the issue from occurring when the button or the SVG inside the button is clicked.

Reply:
I have added the below code along with your suggestion to avoid the showMoreInfo function call.

 `   (e.target as HTMLTableRowElement).tagName === "A" ||
                    (e.target as HTMLTableRowElement).tagName === "svg"`

Reply:
@wackerow  I have added the below code to resolve the issue, but if click on the arrow icon for the right side, then it will not open. So, I have used classList to resolve the issue.

`  if (
              Object.values(classList)?.includes("chakra-button") ||
              Object.values(classList)?.includes("css-1bbj57d")
            ) {
              return
            }`
Should I revert the code, with the classList or not?